services/web/server/tests/integration/conftest.py

```diff
@@ -78,65 +81,53 @@ def webserver_environ(request, simcore_docker_compose, docker_stack) -> Dict[str
         environ['%s_PORT' % name.upper()] = published_port
         # to swarm boundary since webserver is installed in the host and therefore outside the swarm's network
 
-    pprint(environ)
+    pprint(environ) # NOTE: displayed only if error
     return environ
 
 @pytest.fixture(scope='module')
-def app_config(here, webserver_environ) -> Dict:
-    config_file_path = here / "config.yaml"
-    def _recreate_config_file():
-        with app_resources.stream("config/server-docker-dev.yaml") as f:
-            cfg = yaml.safe_load(f)
-            # test webserver works in host
-            cfg["main"]['host'] = '127.0.0.1'
+def webserver_dev_config(webserver_environ: Dict, docker_stack: Dict) -> Dict:
+    """
+        Swarm with integration stack already started
+
+        Configuration for a webserver provided it runs in host
+
+        NOTE: Prefer using 'app_config' instead of this as a fixture
```

Comment:
why this note? is there still a app_config fixture? ok I saw it below.
Why then keep this fixture as a fixture? is there a use-case?

Reply:
I extended the comment